tools/rna_tools/ribotaper/tool_dependencies.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <package name="samtools" version="0.1.19">
+        <repository name="package_samtools_0_1_19" owner="devteam" />
+    </package>
+    <package name="bedtools" version="2.19.1">
+        <install version="1.0">
```

Comment:
Can we get rid of this one and depend on the original one. 
BEDTOOLS_ROOT_DIR should be set with Galaxy 16.01 automatically, hopefully. Otherwise, I will patch the main bedtools recipe.